ICON/PrecipPuzzle.yaml

```
@@ -0,0 +1,21 @@
+sources:
+  mbe2934:
+    driver: netcdf
+    description: Standard AMIP-R2B9 with min_sfc_wind=4.
+    metadata:
+      atmospheric_grid_id: R02B09
+      experiment_id: ICON-AMIP
+      project: PrecipPuzzle
+      simulation_id: mbe2934
+      source_id: https://gitlab.dkrz.de/icon/icon-mpim/-/commit/a9fa36eeb90aa0e2639a4d06fee4a04a00985275
```

```suggestion
      source_id: ICON-ESM
      source_version: https://gitlab.dkrz.de/icon/icon-mpim/-/commit/a9fa36eeb90aa0e2639a4d06fee4a04a00985275
```

Source ID usually just is a readable version of the model name, but I like the idea of (also) adding the actual model version. How about this format?

---

Is there any documentation about the different metadata fields? I also find the distinction between `experiment_id` and `simulation_id` a bit vague. But I am fine with adding a `source_id` and `source_version` :)

---

The idea for experiment_id is something describing the physical meaning (historical / 4x-CO2 / ...), and simulation-ID describing the realization/ensemble member (ngc4026 / r1i1p1f1 in CIMP).

The names are a bit of a mixture of CMIP and DYAMOND and other ideas. I sent you a link to a google doc with the log of the discussions leading to the scheme.
Maybe our STAC team will come up with new ideas anyways...

---

Thanks for the input! For now I went with this approach 👌 

I guess that the actual metadata is still a moving target. However, this way at least all relevant info is available :)